usaspending_api/api_docs/api_documentation/README.md

```diff
@@ -0,0 +1,28 @@
+
+# USASpending API Documentation
+
+
+The Data Accountability Transparency Act (DATA) Act created the USASpending API to advance the transparency of the Federal Government's spending data.  This API's primary use cases are to serve the USASpending.gov website, as well as US citizens as open API. This API's goal is to serve up to date spending data in a quick, accurate manner.
+
+
+
+The USASpending API is in *V2*. V1 endpoints are currently DEPRECIATED. 
+
+
+
+  Although download api endpoints are accessible, PREGENERATED FILES ALREADY EXIST. They are located at:  https://beta.usaspending.gov/#/bulk_download/award_data_archive or https://usaspending.gov/#/bulk_download/award_data_archive
```

Comment:
This doesn't really make sense here. This would make more sense to be part of the download README (if it exists).

Reply:
it does.   And unless dan removed it, it is there

Reply:
Its in the download README. Removing it here,